### DISCUSSION TITLE
i cant enable shift lock (is it even work on this port?)

### DISCUSSION THREAD

**Discussion by Xeone52670v3** [vinegarhq/sober#241] [Q&A]
i downloaded sober yesterday on garuda linux and on this port i cant enable shift lock is it even work on this port if yes why i cant enable it

**Comment by Xeone52670v3** [comment]
i just rebooted it and plug in another keyboard all is fine

**Comment by Bugsbane** [comment]
I thought I had this problem once... until I realized that I'd accidentally turned off the option to use shift lock in the Roblox preferences. *facepalm*

Leaving this here for anyone else looking for a solution to this "problem".

**Comment by SillyMythic** [comment]
I had this issue - shift lock doesn't work when Movement mode is set to Click to Move.
